core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -787,19 +789,41 @@ private[spark] class BlockManager(
   }
 
   /**
+   * Get peer block managers in the system.
+   */
+  private def getPeers(numPeers: Int): Seq[BlockManagerId] = cachedPeers.synchronized {
```

Comment:
cannot not make sense of what `getPeers` means?(it's just my personal confusion....sorry,look forward to receive your reply,thanks) @tdas